ExportAs/ExportAs.py

```diff
@@ -38,8 +38,11 @@ def __init__(self, scriptedPlugin):
     AbstractScriptedSubjectHierarchyPlugin.__init__(self, scriptedPlugin)
 
     self.exportAsAction = qt.QAction(f"Export as...", scriptedPlugin)
+    self.exportTfmAsAction = qt.QAction(f"Export Hardened as...", scriptedPlugin)
     self.menu = qt.QMenu("Plugin Menu")
+    self.menu2 = qt.QMenu("Plugin Menu")
```

Comment:
Here too, please indicate in the name what's different about these two menus.  Give them unique variables names like `self.transformedMenu` and `Transformed Plugin Menu`

Reply:
Right!